Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -618,6 +618,17 @@ Describe "Copy-GcsObject" {
             { Copy-GcsObject -SourceBucket $bucket -SourceObject "test-source" $bucket "test-dest" } |
                 Should Throw 404
         }
+        
+        It "Should fail to read from unaccessable source bucket" {
+            { Copy-GcsObject -SourceBucket "asdf" -SourceObject "test-source" $bucket "test-dest" } |
+                Should Throw 403
+        }
+        
+        It "Should fail to write to unaccessable source bucket" {
+            New-GcsObject $bucket "test-source0" -Contents "test0-$r"
+            { Copy-GcsObject -SourceBucket $bucket -SourceObject "test-source0" "asdf" "test-dest" } |
+                Should Throw 403
+        }
```

Comment:
Need a test to validate `-Force works as expected`. just copy your "should work by name" test, and add another scenario where it fails (already exists) and then works via `-Force`.